### ISSUE TITLE
Message for free users when accepting a shared calendar is inaccurate.

### ISSUE THREAD

**Issue by kibibytium** [tutao/tutanota#4188]
**Description**
Accepting a calendar share as a free user displays the message "This feature can only be ordered after upgrading to a paid subscription..."
This is not accurate since only a premium account is required to accept the share

**Comment by kibhub** [comment]
> Is premium account not paid subscription?

Yes. But it sound from the message that you need a premium account and the Sharing extra feature, which is not necessary
